Tests/ImagineEngineTests/ActorTests.swift

```diff
@@ -420,4 +422,10 @@ final class ActorTests: XCTestCase {
         XCTAssertNil(actor.layer.superlayer)
         XCTAssertNil(actor.scene)
     }
+
+    // TODO: Needed until we find a better way of figuring out the image scale factor on macOS
+    private func scaled(_ size: CGSize) -> CGSize {
```

Comment:
How about making this as a private extension on `CGSize` instead? That way the wording `scaled` makes more sense. Alternatively, naming it `scale` since then it's operation that takes a size and scales it. Nitpicky detail, but anyway 😄

Reply:
Oh and we normally use the `Size` typealias instead of `CGSize`. This to avoid having to import `CoreGraphics` everywhere. Another nitpicky detail 😝 (Gonna add a linting rule for that)

Reply:
Thought of making it an extension but gave up because it only makes sense in the context of these tests. I think the private extension is the way to go indeed 👍